custom_components/tahoma/cover.py

```diff
@@ -1,92 +1,15 @@
-    SUPPORT_OPEN_TILT,
-    SUPPORT_SET_POSITION,
-    SUPPORT_SET_TILT_POSITION,
-    SUPPORT_STOP,
-    SUPPORT_STOP_TILT,
-    CoverEntity,
-)
+from homeassistant.components.cover import ATTR_POSITION, DOMAIN as COVER
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers import entity_platform
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 import voluptuous as vol
 
-from .const import DOMAIN
-from .tahoma_entity import TahomaEntity
-
-ATTR_OBSTRUCTION_DETECTED = "obstruction-detected"
-
-COMMAND_CYCLE = "cycle"
-COMMAND_CLOSE = "close"
-COMMAND_CLOSE_SLATS = "closeSlats"
-COMMAND_DOWN = "down"
-COMMAND_DEPLOY = "deploy"
-COMMAND_MY = "my"
-COMMAND_OPEN = "open"
-COMMAND_OPEN_SLATS = "openSlats"
-COMMAND_SET_CLOSURE = "setClosure"
-COMMAND_SET_CLOSURE_AND_LINEAR_SPEED = "setClosureAndLinearSpeed"
-COMMAND_SET_DEPLOYMENT = "setDeployment"
-COMMAND_SET_ORIENTATION = "setOrientation"
-# io:DiscreteGateOpenerIOComponent
-COMMAND_SET_PEDESTRIAN_POSITION = "setPedestrianPosition"
-
-COMMAND_STOP = "stop"
-COMMAND_STOP_IDENTIFY = "stopIdentify"
-COMMAND_UNDEPLOY = "undeploy"
-COMMAND_UP = "up"
-
-COMMANDS_STOP = [COMMAND_STOP, COMMAND_STOP_IDENTIFY, COMMAND_MY]
-COMMANDS_STOP_TILT = [COMMAND_STOP, COMMAND_STOP_IDENTIFY, COMMAND_MY]
-COMMANDS_OPEN = [COMMAND_OPEN, COMMAND_UP, COMMAND_CYCLE]
-COMMANDS_OPEN_TILT = [COMMAND_OPEN_SLATS]
-COMMANDS_CLOSE = [COMMAND_CLOSE, COMMAND_DOWN, COMMAND_CYCLE]
-COMMANDS_CLOSE_TILT = [COMMAND_CLOSE_SLATS]
-
-COMMANDS_SET_TILT_POSITION = [COMMAND_SET_ORIENTATION]
-
-CORE_CLOSURE_STATE = "core:ClosureState"
-CORE_CLOSURE_OR_ROCKER_POSITION_STATE = "core:ClosureOrRockerPositionState"
-CORE_DEPLOYMENT_STATE = "core:DeploymentState"
-CORE_MEMORIZED_1_POSITION_STATE = "core:Memorized1PositionState"
-CORE_MOVING_STATE = "core:MovingState"
-CORE_OPEN_CLOSED_PARTIAL_STATE = "core:OpenClosedPartialState"
-# io:DiscreteGateOpenerIOComponent
-CORE_OPEN_CLOSED_PEDESTRIAN_STATE = "core:OpenClosedPedestrianState"
-CORE_OPEN_CLOSED_STATE = "core:OpenClosedState"
-CORE_OPEN_CLOSED_UNKNOWN_STATE = "core:OpenClosedUnknownState"
-# io:DiscreteGateOpenerIOComponent
-CORE_PEDESTRIAN_POSITION_STATE = "core:PedestrianPositionState"
-CORE_PRIORITY_LOCK_TIMER_STATE = "core:PriorityLockTimerState"
-CORE_SLATS_OPEN_CLOSED_STATE = "core:SlatsOpenClosedState"
-CORE_SLATE_ORIENTATION_STATE = "core:SlateOrientationState"
-CORE_SLATS_ORIENTATION_STATE = "core:SlatsOrientationState"
-CORE_TARGET_CLOSURE_STATE = "core:TargetClosureState"
-MYFOX_SHUTTER_STATUS_STATE = "myfox:ShutterStatusState"
-
-ICON_LOCK_ALERT = "mdi:lock-alert"
-ICON_WEATHER_WINDY = "mdi:weather-windy"
-
-IO_PRIORITY_LOCK_LEVEL_STATE = "io:PriorityLockLevelState"
-IO_PRIORITY_LOCK_ORIGINATOR_STATE = "io:PriorityLockOriginatorState"
+from custom_components.tahoma.cover_devices.awning import Awning
+from custom_components.tahoma.cover_devices.vertical_cover import TahomaVerticalCover
```

Comment:
Can this be renamed to VerticalCover, without the Tahoma?
```suggestion
from custom_components.tahoma.cover_devices.vertical_cover import VerticalCover
```

Reply:
Good point, I will rename it.